lix/packages/sdk/src/open/openLix.ts

```diff
@@ -142,10 +142,13 @@ async function handleFileChange(args: {
 			neu: args.neuData,
 		})
 		for (const diff of diffs ?? []) {
+			// assume an insert or update operation as the default
+			// if diff.neu is not present, it's a delete operation
+			const value = diff.neu ?? diff.old
```

Comment:
this is only safe to use for things that cannot change, i would do const id = diff.neu?.id ?? diff.old?.id
to prevent accidentally using the value in unexpected ways later on.

Reply:
IMplemented. Thanks